src/transparency/tcp.rs

```diff
@@ -71,7 +71,8 @@ impl Proxy {
         let c = Timeout::new(
             transport::Connect::new(orig_dst, tls),
             self.connect_timeout,
-        );
+        )
+        .named("TCP connection");
```

Comment:
please call this "TCP connect" so that it's less ambiguous.

also the indentation seems pretty weird here.

Reply:
I called it "TCP connection" because "TCP connection timed out" seemed more grammatically correct than "TCP connect timed out". WDYT of "opening TCP connection" so that which end of  the connection this is seems clearer?

And, yeah, I'll fix the indentation, sorry about that.

Reply:
@olix0r I changed both the name and the indentation in cc16f125d72ba58ce3b1360b6133822b63ca4878 :)